docs/source/restore/restore-tables.rst

```diff
@@ -4,63 +4,36 @@ Restore tables
 
 .. note:: Currently, Scylla Manager does not support restoring content of `CDC log tables <https://docs.scylladb.com/stable/using-scylla/cdc/cdc-log-table.html>`_.
 
+.. note::
+   The ``--keyspace`` flag defines base tables which will be restored.
```

Comment:
```suggestion
   The ``--keyspace`` flag defines base tables that will be restored.
```

Reply:
The content of this note is clear, but the usage is not so much. 
Is the user expected to run:
`sctool restore --cluster <id|name> --location [<dc>:]<provider>:<bucket> --snapshot-tag <tag> [flags] ---restore-tables --keyspace`
?
Is adding `--keyspace` required?

I THINK this section should be modified like this:

- (If --keyspace is optional)

_To restore the content of the tables (rows), use the :ref:`sctool restore <sctool-restore>` command with the ``--restore-tables`` flag._

_You **can** add the ``--keyspace`` flag to define the base tables that will be restored.  All Materialized Views and Secondary Indexes of those tables will be automatically recreated in order to ensure the correct data state._

- (If --keyspace is mandatory)

_To restore the content of the tables (rows), use the :ref:`sctool restore <sctool-restore>` command with the ``--restore-tables`` flag._

_**Add** the ``--keyspace`` flag to define the base tables that will be restored.  All Materialized Views and Secondary Indexes of those tables will be automatically recreated in order to ensure the correct data state._